omnigibson/systems/micro_particle_system.py

```diff
@@ -1610,10 +1611,27 @@ def clothify_mesh_prim(cls, mesh_prim, remesh=True, particle_distance=None):
             new_normals = cm.vertex_normal_matrix()
             n_faces = len(cm.face_matrix())
 
-            mesh_prim.GetAttribute("faceVertexCounts").Set(np.ones(n_faces, dtype=int) * 3)
-            mesh_prim.GetAttribute("points").Set(lazy.pxr.Vt.Vec3fArray.FromNumpy(new_vertices))
+            face_vertex_counts = np.ones(n_faces, dtype=int) * 3
+            faces = []
+            i = 0
+            for count in face_vertex_counts:
+                for j in range(count - 2):
+                    faces.append([new_face_vertex_ids[i], new_face_vertex_ids[i + j + 1], new_face_vertex_ids[i + j + 2]])
+                i += count
```

Comment:
You can just not call .flatten() on 1608 and you'll get the array you're building here. You can flatten it later when passing it into 1631. Or you can still flatten it but pass it into trimesh as .reshape(-1, 3)

Reply:
done!